bindings/java/src/org/sleuthkit/datamodel/AbstractContent.java

```diff
@@ -193,6 +193,86 @@ public Content getDataSource() throws TskCoreException {
 
 		return myParent.getDataSource();
 	}
+	
+	/**
+	 * Return whether this content has a Pool above it
+	 * 
+	 * @return true if there is a Pool object in the parent structure
+	 * 
+	 * @throws TskCoreException 
+	 */
+	public boolean isPoolContent() throws TskCoreException {
```

Comment:
Do these need to be public?

Reply:
Oops probably not - I forgot to do a final pass to make things package level.